### PR TITLE
readme: basic setup instructions

### PR DESCRIPTION
**Pull request by shirakaba**
Thought I'd give this a spin but didn't see immediately how to run it locally. Ended up doing these steps, so opening the instructions as a strawman PR.

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -9,6 +9,25 @@ Shadow is a new novel web engine made (almost) entirely in JS from scratch.
 - [Introducing Shadow](https://goose.icu/introducing-shadow/)
 - [Shadow Devlog #1](https://goose.icu/shadow-devlog-1/)
 
+## Contributing
+
+### Installation
+
+```sh
```
Comment (CanadaHonk) [inline readme.md]: This isn't actually needed, just for the weird mostly unusable GTK Node frontend. This installation step can be taken out and the section could just be starting a local HTTP server in the repo dir.

**Review by CanadaHonk — CHANGES_REQUESTED** [review]
Thanks! The installation step isn't actually needed but otherwise is nice.

File: readme.md (hunk 1 above)
Comment (shirakaba) [reply readme.md]: Got it! Have updated accordingly and force-pushed over the previous commit.

**Review by CanadaHonk — APPROVED** [review]
LGTM, thanks!

**Pull request merged**